File: Modul 5/Soal 2/PRAK502-2410817210014-ZAVIERGHANI.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2) {
    return nilai1 - nilai2;
}

int mutlak(int angka) {
    if (angka < 0) {
        return -angka;
    }
    return angka;
}

int main() {
    int a, b, c, d, hasil;

    printf("Masukkan nilai (dipisahkan spasi): ");
    scanf("%d %d %d %d", &a, &b, &c, &d);

    hasil = mutlak(hitung(a, c)) + mutlak(hitung(b, d));
    printf("%d", hasil);

    return 0;
}

File: Modul 5/Soal 2/PRAK502-2410817210014-ZAVIER GHANI.py
def hitung(nilai1, nilai2):
    return abs(nilai1 - nilai2)

print("Masukkan nilai (dipisahkan spasi): ", end="")
x1, y1, x2, y2 = map(int, input().split())
hasil = hitung(x1, x2) + hitung(y1, y2)
print(hasil)
